samples/bluetooth/periodic_sync/src/main.c

```diff
@@ -130,7 +138,11 @@ static void term_cb(struct bt_le_per_adv_sync *sync,
 	printk("PER_ADV_SYNC[%u]: [DEVICE]: %s sync terminated\n",
 	       bt_le_per_adv_sync_get_index(sync), le_addr);
 
+	atomic_dec(&sync_count);
```

Comment:
Sometimes ``term_cb`` function will be called, while ``sync_cb`` is not called before this.
Usually it happen when sync fails exactly after calling ``bt_le_per_adv_sync_create`` (but stnc creation function returns EXIT_SUCCESS).

Reply:
handled in line 306